index.js

```diff
@@ -138,9 +138,12 @@ module.exports.getSetCookies = getSetCookies
 module.exports.setCookie = setCookie
 
 const { parseMIMEType, serializeAMimeType } = require('./lib/web/fetch/data-url')
+const { decompress, decompressStream } = require('./lib/web/fetch/decompress')
 
 module.exports.parseMIMEType = parseMIMEType
 module.exports.serializeAMimeType = serializeAMimeType
+module.exports.decompress = decompress
+module.exports.decompressStream = decompressStream
```

Comment:
I think it is better to use the namespace `util` https://github.com/nodejs/undici/blob/main/docs/docs/api/Util.md

Reply:
Moved it under `core/util.js`.